runtime-sdk/src/modules/accounts/mod.rs

```diff
@@ -857,6 +857,22 @@ impl module::TransactionHandler for Module {
         ctx: &mut C,
         tx: &Transaction,
     ) -> Result<(), modules::core::Error> {
+        // Check whether the transaction is currently valid.
+        let round = ctx.runtime_header().round;
```

Comment:
Do we have a fork ID or something? This is useful, but not complete without domain separation.

Reply:
All transactions are domain-separated by consensus layer chain context and runtime ID.